bot.py

```diff
@@ -3050,7 +3050,7 @@ async def send_long_log_helper(channel, post, depth):
             await client.send_message(channel, "[CONTINUED] " + "```py\n" + post[:max] + "```")
         else:
             await client.send_message(channel, post[:max] + "```")
-        await send_long_log_helper(channel, post[max:], depth + 1)
```

Comment:
You still need the `depth + 1` here to keep track of the number of continues; you can remove the `0` from https://github.com/belguawhale/Discord-Werewolf/pull/53/files#diff-f289368d1ae6d2d58bec9f799832d932R3070 though

Reply:
Thanks for catching this mistake! Fixed it in the new commit.